examples/src/main/scala/io/finch/sse/Main.scala

```diff
@@ -0,0 +1,38 @@
+package io.finch.sse
+
+import java.nio.charset.Charset
+
+import cats.Show
+import com.twitter.concurrent.AsyncStream
+import com.twitter.conversions.time._
+import com.twitter.finagle.Http
+import com.twitter.util.{Future, JavaTimer}
+import io.finch._
+import io.finch.internal.BufText
+
+case class Example(x: Int)
+object Example {
+  implicit val show: Show[Example] = Show.fromToString
+}
+
+object Main {
+  import Example._
+
+  implicit val encodeExceptionAsTextPlain: Encode.EventStream[Exception] = {
```

Comment:
One thing SSE allows is for "comments" to be sent. Perhaps the default serialization of an error could be a comment.